west.yml

```diff
@@ -49,7 +49,7 @@ manifest:
       revision: ef1f9c3d87474ec3570b1f46e91fd4b54a4fb421
     - name: nrfxlib
       path: nrfxlib
-      revision: 150cf3477061d560d43b19617f6cc7e246382794
+      revision: 054312b92e64153cbbcb8ade32a43545efe4eb86
```

Comment:
This is downgrading from master to master~1 , which I am sure you don't want to do.

Reply:
Ok, fixed